Compiled from "ExceptionUtils.java"
public class org.apache.jasper.util.ExceptionUtils {
  public org.apache.jasper.util.ExceptionUtils();
    Code:
       0: aload_0
       1: invokespecial #1                  // Method java/lang/Object."<init>":()V
       4: return

  public static void handleThrowable(java.lang.Throwable);
    Code:
       0: aload_0
       1: instanceof    #2                  // class java/lang/ThreadDeath
       4: ifeq          12
       7: aload_0
       8: checkcast     #2                  // class java/lang/ThreadDeath
      11: athrow
      12: aload_0
      13: instanceof    #3                  // class java/lang/StackOverflowError
      16: ifeq          20
      19: return
      20: aload_0
      21: instanceof    #4                  // class java/lang/VirtualMachineError
      24: ifeq          32
      27: aload_0
      28: checkcast     #4                  // class java/lang/VirtualMachineError
      31: athrow
      32: return

  public static java.lang.Throwable unwrapInvocationTargetException(java.lang.Throwable);
    Code:
       0: aload_0
       1: instanceof    #5                  // class java/lang/reflect/InvocationTargetException
       4: ifeq          19
       7: aload_0
       8: invokevirtual #6                  // Method java/lang/Throwable.getCause:()Ljava/lang/Throwable;
      11: ifnull        19
      14: aload_0
      15: invokevirtual #6                  // Method java/lang/Throwable.getCause:()Ljava/lang/Throwable;
      18: areturn
      19: aload_0
      20: areturn
}
